client/public/js/actions/EventActions.js

```diff
@@ -22,8 +21,16 @@ var EventActions = {
     });
   },
 
+  getAllEventsBySheep: function(sheepId) {
```

Comment:
It's used in Event List for future functionality to show what events you are already subscribed to